cocotb/result.py

```diff
@@ -41,7 +41,7 @@ def raise_error(obj, msg):
         msg (str): The log message.
     """
     exc_info = sys.exc_info()
-    # 2.6 cannot use named access
+    # Python 2.6 cannot use named access
```

Comment:
This comment is useless, cocotb is not supporting python 2.6 and I have no idea what this means anyway

Reply:
(out of scope for this PR though)

Reply:
Yeah, I left it though so that maybe at the point where we remove Python 2 support altogether it provides some context.